pkg/arvo/sys/vane/gall.hoon

```diff
@@ -2,1457 +2,2882 @@
-    ==                                                  ::
-++  prey  (pair volt ffuc)                              ::  privilege
-++  scad                                                ::  foreign connection
-  $:  p/@ud                                             ::  index
-      q/(map duct @ud)                                  ::  by duct
-      r/(map @ud duct)                                  ::  by index
-  ==                                                    ::
-++  scar                                                ::  opaque input
-  $:  p/@ud                                             ::  bone sequence
-      q/(map duct bone)                                 ::  by duct
-      r/(map bone duct)                                 ::  by bone
-  ==                                                    ::
-::                                                      ::
-::  XX a hack, required to break a subscription loop    ::
-::  which arises when an invalid mark crashes a diff.   ::
-::  See usage in ap-misvale.                            ::
-++  misvale-data  (set wire)                            ::  subscrs w/ bad marks
-++  seat                                                ::  agent state
-  $:  misvale/misvale-data                              ::  bad reqs
-      vel/worm                                          ::  cache
-      arms=(map [term path] (unit (pair @ud term)))     ::  ap-find cache
-      mom/duct                                          ::  control duct
-      liv/?                                             ::  unstopped
-      toc/torc                                          ::  privilege
-      tyc/stic                                          ::  statistics
-      ged/gest                                          ::  subscribers
-      hav/vase                                          ::  running state
-      byk/beak                                          ::  update control
-      pyl/(map bone mark)                               ::  req'd translations
-      zam/scar                                          ::  opaque ducts
-  ==                                                    ::
-++  sofa                                                ::  queue for blocked
-  $:  kys/(qeu (trel duct prey club))                   ::  queued kisses
-  ==                                                    ::
-++  stic                                                ::  statistics
-  $:  act/@ud                                           ::  change number
-      eny/@uvJ                                          ::  entropy
-      lat/@da                                           ::  time
-  ==                                                    ::
---                                                      ::
-    ::::::::::::::::::::::::::::::::::::::::::::::::::::::  vane header
-    ::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::
+::  (rest of arvo)
+::
+|%
+::
+::  +internal-gift: synonym for +cuft.
```

Comment:
These should be `+$`, since they're types.  This is also a good candidate for shortening.  The style-guide style would suggest something like this:
```
::  $internal-gift: synonym for $cuft
::  $internal-task: synonym for $cush
::  $agent-action: synonym for $club
::
+$  internal-gift  cuft
+$  internal-task  cush
+$  agent-action   club
```

Also, the comments bother me; I think it's because they just repeat the code itself.  I think the comments should be removed or replaced by something explanatory.

I kinda feel like these should be `=*  internal-gift  cuft`, which would emphasize the fact that they're just aliases.

Reply:
+1 for adding something that is actually explanatory